components/candle/progressionService.ts

```diff
@@ -39,28 +42,38 @@ export class ProgressionService {
         dropIds: string[],
         contractSession: ContractSession,
         userProfile: UserProfile,
+        gameVersion: GameVersion,
```

Comment:
gameVersion is already in the ContractSession.

Reply:
Same for everything down this path.

Reply:
Fixed.